src/GraphQL/ParseGraphQLSchema.js

```diff
@@ -49,7 +49,7 @@ const RESERVED_GRAPHQL_TYPE_NAMES = [
   'DeleteClassPayload',
   'PageInfo',
 ];
-const RESERVED_GRAPHQL_QUERY_NAMES = ['health', 'viewer', 'class', 'classes'];
+const RESERVED_GRAPHQL_QUERY_NAMES = ['health', 'viewer', 'class', 'classes', 'configValue'];
```

Comment:
Why is this called "configValue"? How about "cloudConfig" (feature name) if "config" is too ambiguous?